src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -102,28 +102,15 @@ public async Task<InitializeResult> InitializeAsync(InitializeParams initializeP
         }
 
         [JsonRpcMethod(Methods.InitializedName)]
-        public async Task InitializedAsync()
-        {
-            // Publish diagnostics for all open documents immediately following initialization.
-            var solution = _workspace.CurrentSolution;
-            var openDocuments = _workspace.GetOpenDocumentIds();
-            foreach (var documentId in openDocuments)
-            {
-                var document = solution.GetDocument(documentId);
-                if (document != null)
-                {
-                    await PublishDiagnosticsAsync(document).ConfigureAwait(false);
-                }
-            }
-        }
+        public Task InitializedAsync()
+            => Task.CompletedTask;
```

Comment:
@dibarbet do we need to keep this for contractual reasons, or can it just be removed?

Reply:
Can be removed!

Reply:
Poof.